README.md

```diff
@@ -12,28 +12,23 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
 This service provides an API, which is consumed by other CovidCertificate backend services. It offers an endpoint to allow the decoding and verification of a given Covid Certificate.
 It regularly sends a request to the [Verifier Service](https://github.com/admin-ch/CovidCertificate-App-Verifier-Service) to keep an up-to-date list of trusted DSCs with which Covid Certificates could be signed.
 
-## Simple Verification
+## Local Deployment
 
-In order to allow the independent usage of the Verification-Check-Service without deeper knowledge of the source code, the repository includes a Dockerfile to start the service in a separate container. However, one needs to be in possession of an api-token to access the Verifier-Service.
+In order to allow the independent usage of the Verification-Check-Service without deeper knowledge of the source code, Docker container images are provided on DockerHub (TODO: link). In order for the service to connect to the Verifier-Service, an API token is required. If you intend to deploy your own instance of the Verification-Check-Service, please get in touch with the [BAG](mailto:Covid-Zertifikat@bag.admin.ch) to get a token assigned.
```

Comment:
Could we first try upload it on DockerHub (maybe manually?) and then insert the link here?

Reply:
We could let it deploy from the feature branch once. That way we can test if it works in this repo and get the correct link.